src/main/java/mate/academy/util/InitDatabase.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.util;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import mate.academy.exception.DataProcessingException;
+
+public class InitDatabase {
+    public static void executeInitScript(Connection connection) {
+        String scriptPath = "src/main/resources/init_db.sql";
+        try {
+            String sql = new String(Files.readAllBytes(Paths.get(scriptPath)));
+            String[] queries = sql.split(";");
+            try (Statement statement = connection.createStatement()) {
+                for (String query : queries) {
+                    if (!query.trim().isEmpty()) {
+                        statement.execute(query.trim());
+                    }
+                }
```

Comment:
Use `PreparedStatement` over `Statement` for executing queries, even if they are static and do not have parameters. This is a best practice for security and performance reasons.